speech_to_text.py

```diff
@@ -345,6 +331,14 @@ def load_whisper_model(model_name):
 
 
 if __name__ == "__main__":
+    dotenv.load_dotenv()
```

Comment:
the loading of the dotenv file was moved here so that it doesn't interfere with unit testing, which overrides the runtime environment